lib/metric_collector/cli.py

```diff
@@ -410,12 +410,15 @@ def main():
         global_datapoint[0]['fields']['nbr_threads'] = dynamic_args['nbr_collector_threads']
 
     ### Send results to the right output
-    if dynamic_args['output_type'] == 'stdout':
-        utils.print_format_influxdb(global_datapoint)
-    elif dynamic_args['output_type'] == 'http':
-        utils.post_format_influxdb(global_datapoint, dynamic_args['output_addr'],)
-    else:
-        logger.warn('Output format unknown: %s', dynamic_args['output_type'])
+    try:
+        if dynamic_args['output_type'] == 'stdout':
+            utils.print_format_influxdb(global_datapoint)
+        elif dynamic_args['output_type'] == 'http':
+            utils.post_format_influxdb(global_datapoint, dynamic_args['output_addr'],)
```

Comment:
Should the try/catch only include this line ?
I think right now it includes also the output to STDOUT which would be misleading with the current message. 

Maybe we can put the try/catch inside the `print_format_influxdb` function itself ?

Reply:
I wanted to catch any exceptions with the util functions in general , most exceptions will only happen if something goes wrong with posting to influx anyway.
url is already printed:

logger.info('Sending Datapoint to: %s' % addr)